service/src/main/java/de/adorsys/psd2/sandbox/xs2a/testdata/TestDataMapper.java

```diff
@@ -31,10 +31,13 @@ public TestDataMapper(TestDataService testDataService) {
   /**
    * Maps TestData Account to SpiAccount.
    *
-   * @param account TestData Account
+   * @param account         TestData Account
+   * @param withBalance     optional Query Parameter
+   * @param consentBalances list of balances within consent
    * @return SpiAccount
    */
-  public SpiAccountDetails mapAccountToSpiAccount(Account account) {
+  public SpiAccountDetails mapAccountToSpiAccount(Account account, boolean withBalance,
```

Comment:
What do you think of method overloading for this scenario? So when s.o. does not need balances, he/she does not have to pass this boolean flags with.

For example, there could be another function:
```
/**
   * Maps TestData Account to SpiAccount.
   *
   * @param account TestData Account
   * @return SpiAccount
   */
  public SpiAccountDetails mapAccountToSpiAccount(Account account) {
    return mapAccountToSpiAccount(account, false, null);
  }
```

Reply:
Nice idea, I will implement it that way. Thank you :)